docs/general/builtins/scalar-functions.rst

```diff
@@ -797,6 +797,23 @@ Returns: ``text``
     +--------+
     SELECT 1 row in set (... sec)
 
+.. _scalar-reverse:
+
+``reverse(text)``
+------------------
+Reverses the order of the string.
```

Comment:
I think an empty line is needed after a header for correct rendering.
You generate and check the HTML documentation locally by:
```
./blackbox/bin/sphinx 
open docs/_out/html/index.html
```

```suggestion

Reverses the order of the string.
```

Reply:
Additionally you can use `./blackbox/bin/dev` to constantly and automatically recompile the `.rst` into `html` and see how your changes render, for more details: https://github.com/crate/crate/blob/master/devs/docs/write_docs.rst

Reply:
I did compile the doc locally yesterday and found no issues, 
![image](https://github.com/crate/crate/assets/37985796/3ff4d0f9-dfdb-4710-8158-620c2c62a376)

but I will add the space nonetheless so everything is consistent 👍